sdk/formrecognizer/Azure.AI.FormRecognizer/src/ClientCommon.cs

```diff
@@ -88,5 +89,35 @@ public static RecognizedFormCollection ConvertPrebuiltOutputToRecognizedForms(V2
             }
             return new RecognizedFormCollection(forms);
         }
+
+        public static IReadOnlyList<DocumentWord> GetWords(IReadOnlyList<DocumentPage> pages, IReadOnlyList<DocumentSpan> spans)
+        {
+            var selectedWords = new List<DocumentWord>();
+
+            // Very inefficient implementation for now.
```

Comment:
Second worst implementation I could think of, so we'll definitely replace it with something better.

If we had the guarantee that page words and `spans` are ordered by offset, we could probably make it linear. However, according to Paul:
```cs
The spans should be ordered by reading order, which ideally has incrementing indices. Unfortunately, this is not true in general since we don't yet include selection marks in the overall reading order computation (they are listed at the end). So though the assumption about the index ordering should be true in the future, I would recommend that we sort the spans in the implementation to avoid relying on such behavior.
```
I'm still not 100% sure this affects the `GetWords` implementation, so I still need to think more about it and get some examples.

Reply:
Yeah, this algorithm looks pretty brutal for large documents.  :smile:  Should we make an https://en.wikipedia.org/wiki/Interval_tree out of all the spans and then use that to efficiently intersect them with the words?